src/migrations/20171005123329_transfers.ts

```diff
@@ -8,7 +8,7 @@ exports.up = async (knex: Knex): Promise<any> => {
       sender varchar(66),
       recipient varchar(66),
       token varchar(66) NOT NULL,
-      value numeric NOT NULL CONSTRAINT "nonnegativeTransferValue" CHECK (value >= 0),
+      value VARCHAR(255) NOT NULL,
       "blockNumber" integer NOT NULL CONSTRAINT "positiveTransferBlockNumber" CHECK ("blockNumber" > 0),
```

Comment:
No, but we can't use the constraints to check them because we now have a varchar value.

Probably should file a ticket to put these back into the code that calls the insert.

Reply:
https://app.clubhouse.io/augur/story/8842/add-negative-checks-in-augur-node-sql-constraint